processor/src/fast/basic_block.rs

```diff
@@ -113,11 +113,7 @@ impl FastProcessor {
         // last clock cycle of the BASIC BLOCK ops.
         // For the linked case, check for decorators at an operation index beyond the last operation
         let num_ops = basic_block_node.num_operations() as usize;
-        let decorator_ids = current_forest.decorator_indices_for_op(node_id, num_ops);
-        for &decorator_id in decorator_ids {
-            let decorator = program
-                .get_decorator_by_id(decorator_id)
-                .ok_or(ExecutionError::DecoratorNotFoundInForest { decorator_id })?;
+        for decorator in current_forest.decorators_for_op(node_id, num_ops) {
```

Comment:
Here, and in other similar places, we do not yet check whether we are executing in the debug mode or not, right?